nfl_data_py/__init__.py

```diff
@@ -1000,6 +1001,75 @@ def import_snap_counts(years):
     
     return df
 
+
+
+def import_ftn_data(
+        years, 
+        columns=None, 
+        downcast=True,
+        thread_requests=False
+    ):
+    """Imports FTN charting data
+    
+    FTN Data manually charts plays and has graciously provided a subset of their
+    charting data to be published via the nflverse. Data is available from the 2022
+    season onwards and is charted within 48 hours following each game. This data
+    is released under the [CC-BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/)
+    Creative Commons license and attribution must be made to **FTN Data via nflverse**
+    
+    Args:
+        years (List[int]): years to get weekly data for
+        columns (List[str]): only return these columns
+        downcast (bool): convert float64 to float32, default True
+        thread_requests (bool): use thread pool to read files, default False
+    Returns:
+        DataFrame
+    """
+    
+    # check variable types
+    if not isinstance(years, (list, range)):
+        raise ValueError('Input must be list or range.')
+        
+    if min(years) < 2022:
+        raise ValueError('Data not available before 2022.')
+    
+    if not columns:
+        columns = []
+
+    url = r'https://github.com/nflverse/nflverse-data/releases/download/ftn_charting/ftn_charting_{0}.parquet'
+
+    if thread_requests:
+        with ThreadPoolExecutor() as executor:
+            # Create a list of the same size as years, initialized with None
+            data = [None]*len(years)
+            # Create a mapping of futures to their corresponding index in the data
+            futures_map = {
+                executor.submit(
+                    pandas.read_parquet,
+                    path=url.format(year),
+                    columns=columns if columns else None,
```

Comment:
Simplified with removal of extra logic above.
```suggestion
                    columns=columns,
```